camoco/cli/camoco

```diff
@@ -781,6 +781,11 @@ if __name__ == '__main__':
         help='Perform Gene Ontology Statistics. (default: None)'
     )
     health.add_argument(
+        '--two_tailed_GO',
```

Comment:
Can you change this to be a flag?
```
action='store_true',
default=False
```
This will make the parameter False by default but change it to be true is the flag is included.